bin.ts

```diff
@@ -14,11 +17,12 @@ if (!satisfies(process.version, NODE_VERSION_RANGE)) {
 }
 
 import { Integration, Platform } from './lib/Constants';
-import { run } from './lib/Setup';
```

Comment:
l/m: I wonder if we should add a quick deprecation notice in lib/Setup.ts or smth like a hint not to use that file anymore.